### PR TITLE
Migrate-styles

### PR DESCRIPTION
**Pull request by wolverineks**

### CODE REVIEW

**Comment by wolverineks** [comment]
depends on https://github.com/Emurgo/yoroi-mobile/pull/1944

File: src/AppNavigator.tsx [hunk 1]
```diff
@@ -30,11 +28,13 @@ import {Boundary} from './components'
...
 import {FirstRunNavigator} from './FirstRun/FirstRunNavigator'
 import {CustomPinLoginScreen} from './Login'
+import MaintenanceScreen from './MaintenanceScreen'
+import {defaultNavigationOptions, defaultStackNavigatorOptions} from './navigationOptions'
 import StorybookScreen from './StorybookScreen'
 import {WalletInitNavigator} from './WalletInit/WalletInitNavigator'
 import {WalletNavigator} from './WalletNavigator'
 
-const IS_STORYBOOK = env.getBoolean('IS_STORYBOOK', false)
+const IS_STORYBOOK = !env.getBoolean('IS_STORYBOOK', false)
```
Comment (stackchain) [inline src/AppNavigator.tsx]:
```suggestion
const IS_STORYBOOK = env.getBoolean('IS_STORYBOOK', false)
```

File: src/MaintenanceScreen.tsx [hunk 2]
```diff
@@ -84,23 +63,70 @@ class MaintenanceModal extends React.Component<Props, State> {
...
 
-const MaintenanceScreen = ({intl, initApp}: ScreenProps) => (
-  <MaintenanceModal visible onRequestClose={() => BackHandler.exitApp()} intl={intl} initApp={initApp} />
-)
+  return (
+    <MaintenanceModal
+      visible
+      onRequestClose={() => BackHandler.exitApp()}
+      intl={intl}
+      initApp={(...args) => dispatch(initApp(...args))}
```
Comment (stackchain) [inline src/MaintenanceScreen.tsx]: The `initApp` action does not receive any args I think you can get rid of `...args`

Comment (stackchain) [inline src/MaintenanceScreen.tsx]:
```suggestion
      initApp={() => dispatch(initApp())}
```

**Review by stackchain — APPROVED** [review]

**Pull request merged**